be/src/vec/columns/column_vector.h

```diff
@@ -288,6 +288,12 @@ class ColumnVector final : public COWHelper<ColumnVectorHelper, ColumnVector<T>>
         }
     }
 
+    void set_all_to_zero(size_t num) {
+        for (size_t i = 0; i < num; ++i) {
```

Comment:
may be use memset is better for performance:
`memset(data.data(), 0, sizeof(value_type) * num);`